test/integration/scheduler/volume_binding_test.go

```diff
@@ -715,12 +715,6 @@ func TestVolumeProvision(t *testing.T) {
 			boundPvcs:       []*testPVC{{"pvc-w-canbind", classWait, ""}},
 			provisionedPvcs: []*testPVC{{"pvc-canprovision", classWait, ""}},
 		},
-		"one immediate pvc prebound, one wait provisioned": {
```

Comment:
I think I made this by mistake :(  pvc prebound is not guaranteed, its PV can still be bound by other pvcs.

What makes me a little curious is that in theory, the two pvcs here are created in order, they should get processed in scheduler/controller as the order too. But in fact the order got changed during events generation/consumption, the pv got bound to the pvc that is expected for provision.

Anyway, this is a meaningless case, that should be removed